xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editobject.vm

```diff
@@ -30,6 +30,21 @@
  * Displays a class name without its space prefix. XML-escapes the output.
  *#
 #macro(cleanClassname $classname)$!{escapetool.xml($classname.substring($classname.indexOf('.')).substring(1))}#end
+
+#*
+ * Displays a warning box, with an extra class. Note that the implementation of this macro should be very close to the 
+ * one of the #warning macro defined in the web templates macros.
+ * This is the only place where we use this extraClass feature, so the implementation is contained here.
+ * Note that the exact context this content is rendered in is not fixed, using the XWiki macro 
+ * would have ended up in unstable behaviour.
+ *#
+#macro (warningWithExtraClass $text $extraClass)
```

Comment:
> Now I'm a bit surprised that you don't need that elsewhere.

It's the only place where both:
* I couldn't make it work with XWiki Macro (I tried for an especially long time for this one ^^')
* We needed the `cssClass` argument.

Since it's the only place with this very specific need, I figured it wasn't worth creating and exposing a cssClass argument for all the standard macros. 
The extra CSS class, even when using XWiki Macro, is rarely used (at least in XWiki Standard).

Reply:
> Preferable use name starting with _ if it's not supposed to become public.

My bad, addressed in 
https://github.com/xwiki/xwiki-platform/pull/3555/commits/f75a44cf02b6545d672d8ceab0751fe0656d96bf :+1: